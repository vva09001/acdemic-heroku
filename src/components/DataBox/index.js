import React from "react";
import PropTypes from "prop-types";

const DataBox = ({ title, value }) => {
  return (
    <div className="dataBox">
      <p id="title">{title}</p>
      <p>{value}</p>
    </div>
  );
};

DataBox.propTypes = {
  title: PropTypes.string,
  value : PropTypes.number
};
export default DataBox;
