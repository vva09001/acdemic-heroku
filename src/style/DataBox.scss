$primaryColor: #203e7d;
$secondColor: #0060ae;

.dataBox {
  flex-direction: column;
  text-align: center;
  background: $primaryColor;
  border-radius: 2px;
  p {
    color: #fff;
    margin: 0;
    padding: 5px;
  }
  #title {
    font-size: 12px;
    font-weight: 400;
  }
}
