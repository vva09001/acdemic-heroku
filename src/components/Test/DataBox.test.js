import React from "react";
import { create } from "react-test-renderer";
import DataBox from "../DataBox";

describe("DataBox Component", () => {
  it("Test DataBox component it work", () => {
    const component = create(<DataBox />).toJSON();
    expect(component).toMatchSnapshot();
  });
  exports[`Rerender DataBox component`] = (
    <DataBox title={undefined} value={undefined} />
  );
});
