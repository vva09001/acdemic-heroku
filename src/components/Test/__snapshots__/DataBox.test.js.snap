// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataBox Component Test DataBox component it work 1`] = `
<div
  className="dataBox"
>
  <p
    id="title"
  />
  <p />
</div>
`;
